### PR TITLE
Add Astropy to list of signatories

### PR DESCRIPTION
**Pull request by astrofrog**
The Astropy community has now approved APE 10, which describes the roadmap for releases until 2020, as well as which releases will drop support for Python 2:

https://github.com/astropy/astropy-APEs/pull/13/files

@eteq @taldcroft @kelle - could at least one of you double check this? (especially the dates)

Based on other PRs, I saw the projects are ordered by stars in the project list. Should the order be the same in ``site.js``?

### CODE REVIEW

**Comment by takluyver** [comment]
Thanks, this looks good to me. I'm happy to merge once one of the people you tagged has had a look.

Yes, I think it makes sense for the project order in the timeline to match that in the list of logos. But I'd like to find a better way to display the timeline information entirely - see #46.

**Comment by taldcroft** [comment]
I've checked the dates and they match APE-10 and my expectations.  Note that I did not run this locally.

**Comment by astrofrog** [comment]
Just to confirm, I ran this locally and it all works fine:

screen shot 2016-08-23 at 5 23 11 pm


timeline

**Comment by astrofrog** [comment]
@takluyver - now @taldcroft has confirmed this looks ok, I think this is good to go

File: site.js [hunk 1]
```diff
@@ -136,6 +136,18 @@ $(document).ready(function (){
       {content: '0.9', start: '2017-06-01', end: '2018-06-01', py2:true},
       {content: '0.10', start: '2017-12-01', end: '2018-12-01'},
     ],
+    'Astropy':[
+      {content: '1.0 LTS', start: '2015-02-19', end: '2017-06-01', py2:true},
+      {content: '1.1', start: '2015-12-11', end: '2016-06-23', py2:true},
+      {content: '1.2', start: '2016-06-23', end: '2016-12-01', py2:true},
+      {content: '1.3', start: '2016-12-01', end: '2017-06-01', py2:true},
```
Comment (eteq) [inline site.js]: 1.3 "start" (and the "end" of 1.2) should probably be ``'2016-12-16'``, as that's what we currently have as the release date on https://github.com/astropy/astropy/wiki/Release-Calendar

**Comment by eteq** [comment]
Looks good to me too, @astrofrog, except for the minor tweak I just suggested above.

File: site.js (hunk 1 above)
Comment (astrofrog) [inline site.js]: Thanks, fixed!

**Comment by Carreau** [comment]
It's great ! Have been working hard to get some change into PyPI legacy, right now we are hitting a PgSQL bottle neck, so if anyone as experience with SQL to optimize queries...

**Comment by takluyver** [comment]
Thanks, and welcome on board :-)

**Pull request merged**